src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/results.html.twig

```diff
@@ -31,9 +31,30 @@
                         <td class="text-center">
                             <span class="label {{ css_class }}">{{ result.status_code|default('n/a') }}</span>
                         </td>
-                        <td class="nowrap">{{ result.ip }}</td>
-                        <td>{{ result.method }}</td>
-                        <td class="break-long-words">{{ result.url }}</td>
+                        <td>
+                            <span class="nowrap">{{ result.ip }}</span>
+                            {% if request.session is not null %}
+                                <a href="{{ path('_profiler_search_results', request.query.all|merge({'ip': result.ip, 'token': result.token})) }}" title="Search">
```

Comment:
:+1: which one do you propose? Currently we don't use "filter" anywhere, so I sticked with "search".